index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/styles.css" />
+  <title>My Portifolio</title>
+</head>
+
+<body>
+  <nav class="flexr container">
+    <a href="#" class="link">
+      <img src="https://arsenethierry-dev.vercel.app/images/logo-light.svg" width="90px" height="60px" alt="github icon" />
+    </a>
+    <img src="/images/menu button.png" width="18px" height="12px" id="menu" alt="menu icon" />
+    <ul>
+      <li><a href="about.html" class="link">Portfolio</a></li>
+      <li><a href="About.html" class="link">About</a></li>
+      <li><a href="contact.html" class="link">Contact</a></li>
+    </ul>
+  </nav>
```

Comment:
- [ ) As the previous reviewer required, kindly wrap these elements into `header` tag. The hamburger button and the ul list should be wrapped into the `nav` tag.